chef/.kitchen.yml

```diff
@@ -1,14 +1,15 @@
 ---
 driver:
   name: vagrant
-  provider: vmware_fusion
 
 provisioner:
   name: chef_solo
+  chef_omnibus_url: "https://gist.githubusercontent.com/netshade/28800461038c924b74a1/raw/a453b4758ccb9b3d20f95d5465f0754c0fa6ffc3/install.sh"
```

Comment:
Interesting. So it's currently on a different domain because this isn't in master yet?